docs/modules/project-goals/pages/index.adoc

```diff
@@ -1,9 +1,125 @@
-= Project Goals
+//Refs
+:fn-sbomer-ref: footnote:sbomer[https://github.com/project-ncl/sbomer[SBOMer]]
+:fn-trustify-ref: footnote:trustify[https://github.com/trustification/trustify[trustify]]
 
-include::ROOT:partial$wip.adoc[]
+= Project Goals
 
 With the Mequal project we have 3 high-level goals we’d like to achieve:
 
-. Validation
-. Evaluation
-. Policy Authoring
\ No newline at end of file
+*Validation*
+
+We would like to identify if a manifest is up-to-standard not just in its schema representation, but beyond, where we can also identify problems where the standard schema checks of different manifest formats just wouldn't be able to identify them. We'd also like to validate certain representations of a manifest depending on the context or what type of manifest it is.
```

Comment:
I would add here that validation is about ensuring the manifest is "correct" according to some rules. And maybe something about the business value of the manifest which cannot be checked with just a schema validation.

Reply:
Done, added another paragraph to reflect this.